frontend/src/components/TaskTableUnrated.module.scss

```diff
@@ -0,0 +1,11 @@
+@import '../shared.scss';
+
+.tableContainer {
+  margin-bottom: 50px;
+}
```

Comment:
This might be better defined by the user of these task tables

Also, a trailing newline would be nice for consistency, it seems that your setup leaves that out from scss files.